cmd/create/cluster/cmd.go

```diff
@@ -287,6 +288,13 @@ func init() {
 		"A proxy URL to use for creating HTTPS connections outside the cluster.",
 	)
 
+	flags.StringVar(
```

Comment:
Maybe this should be `flags.StringSliceVar`, that way you can do both comma-separated and multiple flags:
```
$ rosa ... --no-proxy foo --no-proxy bar
```
as well as
```
$rosa ... --no-proxy foo,bar
```
and they will automatically be stored in a `[]string`